src/main/java/it/gov/pagopa/logextractor/util/external/filestorage/FileClientService.java

```diff
@@ -0,0 +1,40 @@
+package it.gov.pagopa.logextractor.util.external.filestorage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.OutputStream;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import it.gov.pagopa.logextractor.util.external.IStorageService;
+
+@Profile("dev2")
+@Service
+public class FileClientService implements IStorageService{
+	@Value("${localstorage.basepath:c:\\tmp\\}")
+	private String basePath;
+
+	@Override
+	public OutputStream uploadStreamV2(String keyName) {
+		try {
+			return new CustomFileOutputStream(basePath+keyName);
+		} catch (FileNotFoundException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

Comment:
Sarebbe preferibile eliminare la parte di printStackTrace e loggare con un log (ERROR o WARING): https://sonarcloud.io/project/security_hotspots?id=pagopa_pn-logextractor-be&pullRequest=79&resolved=false&types=SECURITY_HOTSPOT